test/boolean_test.cpp

```diff
@@ -296,6 +296,67 @@ TEST(Boolean, SplitByPlane60) {
               splits.second.GetProperties().volume, 1e-5);
 }
 
+TEST(Manifold, ConvexConvexMinkowski) {
+  bool oldDeterministic = ManifoldParams().deterministic;
+  ManifoldParams().deterministic = true;
+  float offsetRadius = 0.1f;
+  float cubeWidth = 2.0f;
+  Manifold sphere = Manifold::Sphere(offsetRadius, 20);
+  Manifold cube = Manifold::Cube({cubeWidth, cubeWidth, cubeWidth});
+  Manifold sum = cube.MinkowskiSum(sphere);
+  EXPECT_NEAR(sum.GetProperties().volume,
+              powf(cubeWidth + (2 * offsetRadius), 3.f), 0.06f);
+  EXPECT_EQ(sum.Genus(), 0);
+  Manifold difference = Manifold::Cube({cubeWidth, cubeWidth, cubeWidth})
+                            .MinkowskiDifference(sphere);
+  EXPECT_NEAR(difference.GetProperties().volume,
+              powf(cubeWidth - (2 * offsetRadius), 3.f), 1e-5);
```

Comment:
We still need to compare the analytic surface area too, or we might have shards that are connected in such a way as to not affect genus.

Reply:
Hrmm, these are good shouts, but the ConvexConvex case here is just a single Hull operation where the vertices of B are duplicated for all the vertices of A... I don't think we need to go to such a great length testing a simple case of the QuickHull algorithm, do we?

Likewise, I don't know of any simple formulas for the NonConvex/Convex and NonConvex/NonConvex cases... so perhaps it would be simpler to just use magic numbers from my machine where it appears to have favorable rounding characteristics?

Reply:
Alright, fair point.